lib/rules/no-absolute-internal-imports.js

```diff
@@ -18,6 +18,10 @@ const getRelativePath = filePath => {
 const getHasError = (filePath, importSource) => {
     const relativeFileLocation = getRelativePath(getWithoutLastSegment(filePath))
 
+    if (relativeFileLocation.split('/').length === 1) {
```

Comment:
shouldn't this check be after that check: https://github.com/pmedianetwork/eslint-config/pull/70/files#diff-a94f03f5dc304c6143eed931f1406ab4R25 ❓

Reply:
It will be :D
https://github.com/pmedianetwork/eslint-config/pull/70/commits/12292f7532f48d4fc1070db7194ad8256992c994